index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 
 <head>
-  
+
```

Comment:
@SaptakS going through the other files, I see that there are whitespaces and unnecessary new lines at many places. Thanks for pointing this out. Shall we add an issue for this or is it minor? I can work on removing them if there's a need.